conda/activate.py

```diff
@@ -71,10 +87,16 @@ def __init__(self, shell, arguments=None):
             self.command_join = ';\n'
 
             self.unset_var_tmpl = 'unset %s'
-            self.export_var_tmpl = 'setenv %s "%s"'
+            self.unexport_var_tmpl = 'unsetenv %s'
```

Comment:
I see no reason to introduce `unexport_var_tmpl`. With your changes you effectively just replaced `unset_var_tmpl` with that. The `unset_var_tmpl` is only used to unset environment variables. If you think `unsetenv` should be used for `csh`, then just you that in `unset_var_tmpl`.

Reply:
While I agree with this I chose to add it as it makes the solution more distinct in what it's actually unsetting - but alright - I'll revert.